5주차/20922/java_20922_panda.java

```diff
@@ -0,0 +1,40 @@
+package algorithm_study.week5;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.Queue;
+import java.util.StringTokenizer;
+
+public class java_20922_panda {
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st = new StringTokenizer(br.readLine());
+        int N = Integer.parseInt(st.nextToken());
+        int K = Integer.parseInt(st.nextToken());
+
+        int max = 0;
+        Map<Integer, Integer> counts = new HashMap<>();
+        Queue<Integer> sub = new LinkedList<>();
+        st = new StringTokenizer(br.readLine());
+        for (int i = 0; i < N; i++) {
+            int num = Integer.parseInt(st.nextToken());
+            sub.add(num);
+            int count = counts.getOrDefault(num, 0);
+            counts.put(num, ++count);
+            if (count > K) {
```

Comment:
코테랑 전혀 상관 없는 이야기지만 이 쪽이 로직 파악하기 더 쉬울 것 같아요 ㅎㅎ (가독성 측면에서!)
```suggestion
            int count = counts.getOrDefault(num, 0) + 1;
            counts.put(num, count);
            if (count > K) {
```